[Question]:  Cannot find module '@typegoose/auto-increment'

### Checked Existing

- [X] I have checked the repository for duplicate issues.

### What would you like to know?

Regardless of whether I use the finished Docker version or via source, I get the error: 

```
[10:51:25] [INFO]: Loading config
[10:51:25] [INFO]: Config loaded, checking integrity
node:internal/modules/cjs/loader:1228
  throw err;
  ^
Error: Cannot find module '@typegoose/auto-increment'
Require stack:
- /home/node/app/dist/models/maker.js
- /home/node/app/dist/database.js
- /home/node/app/dist/middleware/maker.js
- /home/node/app/dist/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/home/node/app/dist/models/maker.js:8:26)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
    at Module.require (node:internal/modules/cjs/loader:1311:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/node/app/dist/models/maker.js',
    '/home/node/app/dist/database.js',
    '/home/node/app/dist/middleware/maker.js',
    '/home/node/app/dist/server.js'
  ]
}
Node.js v20.17.0
``` 

However, the module itself seems to be installed. 

Can anyone help me with this?

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/PretendoNetwork/rpg-maker-fes#readme",
   "dependencies": {
     "@aws-sdk/client-s3": "^3.303.0",
+    "@typegoose/auto-increment": "^3.0.0",
     "colors": "^1.4.0",
     "crc-32": "^1.2.2",
     "dotenv": "^16.0.3",
@@ -35,7 +36,6 @@
     "parse-multipart-data": "^1.5.0"
   },
   "devDependencies": {
-    "@typegoose/auto-increment": "^3.0.0",
     "@types/express": "^4.17.17",
     "@types/fs-extra": "^11.0.1",
     "@types/morgan": "^1.9.4",

